.github/workflows/release.yml

```diff
@@ -43,10 +43,6 @@ jobs:
 
           npm install -g npm
 
-          npm ci
-          npm --no-git-tag-version version ${{ github.ref_name }}
-          npm run build
-
-          docker buildx build --push --platform linux/arm64/v8,linux/amd64 --build-arg VER=${{ github.ref_name }} --tag ghcr.io/defenseunicorns/pepr/controller:${{ github.ref_name }} .
+          docker buildx build --push --platform linux/arm64/v8,linux/amd64 --build-arg PEPR_BUILD_VERSION=${{ github.ref_name }} --tag ghcr.io/defenseunicorns/pepr/controller:${{ github.ref_name }} .
 
           PEPR_VERSION=${{ github.ref_name }} npm publish --provenance --access public
```

Comment:
I am not sure if `PEPR_VERSION` does anything here. I can't find any other references to it in the project.